app/models/spotlight/blacklight_configuration.rb

```diff
@@ -252,11 +252,11 @@ def add_exhibit_specific_fields(config)
     def add_exhibit_tags_fields(config)
       # rubocop:disable Style/GuardClause
       unless config.show_fields.include? :exhibit_tags
-        config.add_show_field :exhibit_tags, field: Spotlight::SolrDocument.solr_field_for_tagger(exhibit), link_to_search: true
+        config.add_show_field :exhibit_tags, field: ::SolrDocument.solr_field_for_tagger(exhibit), link_to_search: true
```

Comment:
Instead of using `::SolrDocument` directly, we'll need to use `blacklight_config.document_model`, as that's a configurable class too.